README.md

```diff
@@ -139,10 +135,10 @@ VMware ESXi:
 - 6.0 u2
 - 6.5
 
-Docker: 1.9 and higher
+Docker: 1.9 and higher (Recommended 1.13/17.03 and above to use managed plugin)
 
```

Comment:
Shouldn't this be like, "Docker versions earlier than 1.13 aren't supported any more as the vDVS is only supported as a managed plugin (Docker versions 1.13/17.03 and later).